packages/ui-core/src/components/table/sort-arrows.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Icon } from '@keen.io/icons';
+import { ArrowsType } from '../../types';
+import { UpArrow, DownArrow, Container } from './table.styles';
+
+type Props = {
+  type: ArrowsType;
```

Comment:
IMO the component should accept:
```
sortMode: 'ascending' | `descending`;
```
Based on that render a proper User interface - right now there is a separate type that connects UI with sort - which is disruption of the domain.